tools/libdeflate/Makefile

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright (C) 2022 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libdeflate
+PKG_VERSION:=1.15
+PKG_RELEASE:=$(AUTORELEASE)
+
+PKG_SOURCE_URL:=https://github.com/ebiggers/libdeflate.git
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_VERSION:=020133854ff73b8506fe59f92a9b5b622d360716
+PKG_MIRROR_HASH:=122feff4543541b547dc89e832adf262c81911ae1acbccdc591f0353a85b600a
+
+include $(INCLUDE_DIR)/host-build.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+CMAKE_HOST_OPTIONS += \
```

Comment:
Probably makes more sense to build static.

Reply:
wait this is already default.

Reply:
It seems to work 'just fine' here obviously the .tar.xz isn't mirrored by openwrt yet? but github only offers a .tar.gz, though openwrt may be smart tod ownload the file automagically as tar.xz?/

As for cmake; I just took the hint from a bunch of others from the 'tools' Makefiles, e.g. firmware-utils, all they do is `include $(INCLUDE_DIR)/cmake.mk` so figured that's enough to satisfy the dependency?

Reply:
Not for tools

Reply:
so why is 'firmware-utls' under tools? and how do I add the dependency? :)

Reply:
I linked to a commit that shows how.